app/.prettierignore

```diff
@@ -1,3 +1,6 @@
 # Ignoring generated files
 .next/
 node_modules/
+
+# Ignore USWDS static assets
+public/uswds
```

Comment:
nit: all of `public/` could probably be ignored from code formatting

Reply:
I'm going to ignore this nit because next-i18next / react-i18next puts the translation files in `public/locales`.